torchvision/utils.py

```diff
@@ -164,11 +164,11 @@ def draw_bounding_boxes(
 
     """
     Draws bounding boxes on given image.
-    The values of the input image should be uint8 between 0 and 255.
+    The image values should be in the uint8 or float32 that range between 0 and 255.
```

Comment:
Here and in the other places:

```suggestion
    The image values should be uint8 in [0, 255] or float in [0, 1]
```